.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build:
+    if: github.repository == 'jazzband/django-payments'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U setuptools twine wheel
+
+      - name: Build package
+        run: |
+          python setup.py --version
+          python setup.py sdist --format=gztar bdist_wheel
+          twine check dist/*
+
+      - name: Upload packages to Jazzband
```

Comment:
This pushes into a `jazzband` repository, rather than public PyPI.

Is this then mirrored or something alike?

Reply:
Yeah, this is part of Jazzband's semi-automatic release process, where tagging a release in the repo triggers the release to the Jazzband package index (basically a staging package index), which in turn asks the project lead(s) by email to confirm the release before pushing to PyPI. More about this can be found in the [releases docs](https://jazzband.co/about/releases).  Since you're the project lead this adds a bit more overhead for you but  removes the bottleneck of package releases in case another project lead takes over in the future.

Reply:
I  should note that almost all Jazzband projects have been configured for that and it's strongly recommended to simplify the release process.

Reply:
Makes sense.

The docs still mention Travis; might want to update that.

Reply:
Yeah, on the TODO list ;)